src/sentry/integrations/vercel/urls.py

```diff
@@ -12,5 +12,9 @@
         VercelExtensionConfigurationView.as_view(),
         name="sentry-extensions-vercel-configure",
     ),
-    url(r"^delete/$", VercelUninstallEndpoint.as_view(), name="sentry-extensions-vercel-delete"),
+    url(
+        r"^delete/$",
```

Comment:
@MeredithAnya maybe worth adding a comment how we will use a different route in the future